packages/web/config/chain-infos.ts

```diff
@@ -3,18 +3,25 @@ import { Bech32Address } from "@keplr-wallet/cosmos";
 import { createKeplrChainInfos, SimplifiedChainInfo } from "./utils";
 
 const IS_TESTNET = process.env.NEXT_PUBLIC_IS_TESTNET === "true";
+const RPC_OVERWRITE = process.env.NEXT_PUBLIC_RPC_OVERWRITE;
```

Comment:
Could we prepend `OSMOSIS_` to each of these constants to clarify usage?

Reply:
done~